src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +18,36 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        String str = "";
+        try {
+            File file = new File(fileName);
+            FileReader fileReader = new FileReader(file);
+            BufferedReader reader = new BufferedReader(fileReader);
+            while (reader.ready()) {
+                str += reader.readLine();
+            }
+            reader.close();
+        } catch (FileNotFoundException e) {
+            System.out.println(e);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        str = str.replaceAll("[^\\s a-z A-Z]", "");
+        String[] words = str.split(" ");
+        List<String> list = new ArrayList<>();
+        for (int i = 0; i < words.length; i++) {
+            if (words[i].toLowerCase().startsWith("w")) {
+                list.add(words[i].toLowerCase());
+            }
+        }
+        if (list.isEmpty()) {
```

Comment:
can we crate a solution without this check?

Reply:
yes,we can. Sorry, it was late night, when I finished hw, so I add this check. But now I see, that we can figure without it.